### PR TITLE
CDR-1401 Add Robot tests with Basic and OAUTH2 auth types

### PR DESCRIPTION
**Pull request by vladislavploaia**

### CODE REVIEW

File: tests/keycloak/import/ehrbase-realm-exported.json [hunk 1]
```diff
@@ -0,0 +1,2143 @@
...
+      "containerId" : "ca3e0ff6-5282-4d18-b860-8d28d7ac1b86",
+      "attributes" : { }
+    }, {
+      "id" : "e47d350d-a951-4ddb-9923-657b981ce38f",
+      "name" : "tenant-admin",
+      "composite" : false,
+      "clientRole" : false,
+      "containerId" : "ca3e0ff6-5282-4d18-b860-8d28d7ac1b86",
+      "attributes" : { }
+    }, {
```
Comment (alexlehn) [inline tests/keycloak/import/ehrbase-realm-exported.json:87]: we should double check what roles are needed ;).

File: tests/docker-compose-int-test.yml [hunk 2]
```diff
@@ -31,23 +30,28 @@ services:
...
+    networks:
+      - ehrbase-net
 
   #
   # Integration tests will be ready after ehrbase is healthy.
   # Will run `runRobotTests`
   #
   ehrbase-integration-tests:
-    image: ${EHRBASE_INTEGRATION_TEST_IMAGE:-ehrbase/integration-tests:latest}
+    image: ${EHRBASE_INTEGRATION_TEST_IMAGE:-ehrbase/integration-tests:latest-feature}
```
Comment (alexlehn) [inline tests/docker-compose-int-test.yml:49]: should this be changed back to `:latest`

**Review by alexlehn — COMMENTED** [review]
The Keycloak is now running for all tests, also in case it is not needed. But it seems it does not affect the tests that much.

File: tests/keycloak/import/ehrbase-realm-exported.json (hunk 1 above)
Comment (vladislavploaia) [reply tests/keycloak/import/ehrbase-realm-exported.json]: @alexlehn removed all others roles + users and kept the needed only.

File: tests/docker-compose-int-test.yml (hunk 2 above)
Comment (vladislavploaia) [reply tests/docker-compose-int-test.yml:49]: @alexlehn will change back to **:latest** once pipeline is passed after cleaning up the roles and users from Keycloak.
Before merging to develop, will change to latest tag.

**Review by alexlehn — APPROVED** [review]
@vladislavploaia please squash & merge ;)

**Pull request merged**